### PR TITLE
PMM-9737 kill backlog

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
created a test for case PMM-T1051 to verify PMM Settings page is opened from Home dashboard
added a test for case PMM-T1087 to verify adding PostgreSQL remote instance without postgres database
and a test for case PMM-T969 to verify pmm-client logs when incorrect public address is set

T1051 - OK
https://pmm.cd.percona.com/view/all/job/pmm2-ui-tests/14339/
image

T1087 - OK
https://pmm.cd.percona.com/view/all/job/pmm2-ui-tests/14633/
image

T969 - OK
https://pmm.cd.percona.com/job/pmm2-dbaas-e2e-tests/866/
image

### CODE REVIEW

File: tests/verifyRemoteInstances_test.js [hunk 1]
```diff
@@ -265,3 +266,37 @@ Data(metrics).Scenario(
     await grafanaAPI.waitForMetric(current.metricName, { type: 'service_name', value: current.serviceName }, 10);
   },
 );
+
+Scenario(
+  'PMM-T1087 Verify adding PostgreSQL remote instance without postgres database @instances',
```
Comment (puneet0191) [inline tests/verifyRemoteInstances_test.js:271]: I think you did the opposite of what was the expectation for this test case, we have to verify that users can add PostgreSQL remote instances even without having the default Postgres database, what you did is actually just verifying if the user-provided DB exists or not and giving back the error.

File: tests/upgradePMM_test.js [hunk 2]
```diff
@@ -977,3 +977,19 @@ if (versionMinor >= 23) {
     },
   ).retry(1);
 }
+
+Scenario(
+  'PMM-T1051 - Verify PMM Settings page is opened from Home dashboard @menu',
```
Comment (puneet0191) [inline tests/upgradePMM_test.js:982]: why did you add this test to upgrade test? and if we want to add this scenario for upgrade verifications, the tags don't work at all in that scenario, @menu tag tests should be inside the menu tests

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -111,6 +111,35 @@ services:
     networks:
       - pmm-network
 
+  # it needs for deleting postgres db for test T1087
+  postgresremovedefaultdb:
```
Comment (puneet0191) [inline docker-compose.yml:115]: @ElenaFedichkina could you not have a separate compose file for this specific case, you could always start the containers, in your test and kill them post that.

File: docker-compose.yml [hunk 4]
```diff
@@ -111,6 +111,35 @@ services:
...
+
+  postgresnodb:
+    image: ${POSTGRES_IMAGE:-perconalab/percona-distribution-postgresql:13.3}
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    ports:
+      - "5435:5432"
```
Comment (puneet0191) [inline docker-compose.yml:135]: or change the port to something like 6435 to avoid conflict with other tests

**Review by puneet0191 — APPROVED** [review]
good stuff here, thank you

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**